### PR TITLE
Fix `sticky-file-header` positioning

### PR DESCRIPTION
**Pull request by fregante**
- Follows https://github.com/refined-github/refined-github/pull/5717

## Before

Screen Shot 9

# PR (Native)

https://github.com/refined-github/refined-github/pull/5460/files

Screen Shot 6

# Gist

https://gist.github.com/sindresorhus/a39789f98801d908bbc7ff3ecc99d99c

Screen Shot 4

# Compare

https://github.com//refined-github/refined-github/compare/22.2.13...22.2.22#files_bucket

Screen Shot 3

# Commit

https://github.com/refined-github/refined-github/commit/1d9dd9f0208c327351512beb6d6a22279bc47807

Screen Shot 2

# File editing 

https://github.com/refined-github/refined-github/edit/main/readme.md

Screen Shot 11

# Commit with notification bar

before

# Blame

https://github.com/refined-github/refined-github/blame/243e51d786e4771d313091b94cec826ff86becc9/source/features/index.tsx#L326-L327

Screen Shot 1

### CODE REVIEW

**Comment by fregante** [comment]
Also hotfixed

**Comment by fregante** [comment]
Also broken on https://gist.github.com/sindresorhus/a39789f98801d908bbc7ff3ecc99d99c

Before and after
Screen Shot 12

Screen Shot 13

**Comment by kidonng** [comment]
I hotfixed it for Gists. And notice this is broken on [compare pages](https://github.com//refined-github/refined-github/compare/22.2.13...22.2.22#files_bucket) as well 😅

image

File: source/features/sticky-file-header.css [hunk 1]
```diff
@@ -1,9 +1,24 @@
 /* `.sticky-file-header` excludes native sticky header */
-.file-header:not(.sticky-file-header) {
+/* `blob-editor` excludes isEditFile */
+:not(blob-editor) > .file-header:not(.sticky-file-header) {
 	position: sticky;
 	top: 60px;
```
[the remark this answers is not in the document]
Comment (kidonng) [reply source/features/sticky-file-header.css:5]: How about this:

1. Revert my last PR
2. Add a **new** rule for commit pages
3. (Optionally?) narrow old selectors

**Comment by fregante** [comment]
Someone please mention me on https://github.com/refined-github/sandbox/commit/c8cc222ee49e622cf04cbc128af3e46e616681fa so I can test it

**Comment by fregante** [comment]
It workenstein. Check the first post for updated screenshots

**Comment by fregante** [comment]
Why did GitHub close it? mmm

**Comment by fregante** [comment]
The default action for GitHub is "Close and comment." Neat.

**Comment by fregante** [comment]
@kidonng I noticed that your current hotfix works on the file editor page while my code does not. I restored the z-index fix only for that page. How does it look?

**Comment by fregante** [comment]
Blame bug, fixed

https://github.com/refined-github/refined-github/blame/243e51d786e4771d313091b94cec826ff86becc9/source/features/index.tsx#L326-L327

Screen Shot 2

**Pull request merged**